src/CMakeLists.txt

```diff
@@ -258,8 +259,7 @@ if(NCNN_THREADS)
     if(TARGET Threads::Threads)
         target_link_libraries(ncnn PUBLIC Threads::Threads)
     endif()
-
-    if(NCNN_SIMPLEOMP OR NCNN_SIMPLESTL)
+    if(NCNN_SIMPLEOMP OR NCNN_SIMPLESTL OR NCNN_SIMPLEMATH)
```

Comment:
simplemath does not imply pthread linking, does it ?

Reply:
Done